pkgs/applications/video/gpu-screen-recorder/gpu-screen-recorder-gtk.nix

```diff
@@ -13,11 +13,11 @@
 
 stdenv.mkDerivation {
   pname = "gpu-screen-recorder-gtk";
-  version = "3.2.5";
+  version = "3.7.6";
 
   src = fetchurl {
-    url = "https://dec05eba.com/snapshot/gpu-screen-recorder-gtk.git.r175.cfd18af.tar.gz";
-    hash = "sha256-HhZe22Hm9yGoy5WoyuP2+Wj8E3nMs4uf96mzmP6CMqU=";
+    url = "https://dec05eba.com/snapshot/gpu-screen-recorder-gtk.git.r277.cd777c1.tar.gz";
```

Comment:
Can we utilise this? https://git.dec05eba.com/gpu-screen-recorder-gtk

Reply:
if you talk about fetchgit, there are two problems:
1) this link is not recognized as repo(both nix error and `git clone` command)
2) in `refs/tags` there are no tags and we would still rely on commits
ps. maybe I'm missing something, but task for now is just to bump version to match `gpu-screen-recorder` in nix repos

Reply:
Works for me.
![image](https://github.com/NixOS/nixpkgs/assets/50095635/6746b158-0a3c-4a5d-b614-4d1139fe81c3)

Reply:
> Works for me. ![image](https://private-user-images.githubusercontent.com/50095635/342782821-6746b158-0a3c-4a5d-b614-4d1139fe81c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzMjk3NjUsIm5iZiI6MTcxOTMyOTQ2NSwicGF0aCI6Ii81MDA5NTYzNS8zNDI3ODI4MjEtNjc0NmIxNTgtMGEzYy00YTVkLWI2MTQtNGQxMTM5ZmU4MWMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI1VDE1MzEwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjMWJmYzE2ZWRiYTY5YTcwN2FiYTk0MDIwY2ViNWExZWVhZjQ5MGNlYjAyNDAxNzc3ZmQ0NjJmZDBjMmZlOWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MIaGNB2Aet4gos85_3EjiGuN07IQcwBhKIlSGUmv4Ns)

Oh, I see... My bad, I'll fetchgit than

Reply:
done